docs/en/_navbar.md

```diff
@@ -1,3 +1,4 @@
 - Translations
   - [:uk: English](/)
+  - [:ro: Romanian](/ro/)
```

Comment:
replace `:ro:` -> `:romania:`
then the flag will be displayed normally
:ro: -> :romania: